.github/workflows/lint.yml

```diff
@@ -0,0 +1,28 @@
+name: Lint
+
+on: push
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install yamllint
+        run: pip install yamllint
+
+      - name: Lint YAML files
+        run: make lint
+
+  syntax_check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Check playbooks with --syntax-check
+        run: make syntax-check
```

Comment:
is there a reason why are you using make instead of github pipeline directly?

Reply:
I'm using make so that we ensure consistency of running the command. I don't want people running the command one way locally and then a different way on the CI.